src/main/java/higtools/utils/HIGUtils.java

```diff
@@ -6,10 +6,17 @@
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
 import meteordevelopment.meteorclient.utils.world.TickRate;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.IntStream;
 
 import static meteordevelopment.meteorclient.MeteorClient.mc;
 
 public class HIGUtils {
+    private HIGUtils() {
+    }
```

Comment:
Since it's a Utils class, with all static methods, you shouldn't be able to call its constructor anywhere in the code.
So I added a private one to shadow the default public one.

Reply:
Oh true, my bad lol